x-pack/legacy/plugins/file_upload/public/util/http_service.js

```diff
@@ -17,11 +14,10 @@ export async function http(options) {
     });
   }
   const url = options.url || '';
-  const headers = addSystemApiHeader({
```

Comment:
FYI you should be passing the `asSystemRequest` option to http.fetch if you still need this. That said, I'm not sure this request should be a system request. That flag is mostly used to bypass logic like session expiration or unauthorized responses that should redirect to the login page.

Reply:
Good to know. I don't think we'd want to treat it as a system request since eventually we'll want it to redirect to login if needed (i.e.- not bypass the logic that checks auth, etc.). So, sounds like the changes in place make sense?